.github/workflows/test.yml

```diff
@@ -9,12 +9,12 @@ on:
   workflow_call:
 
 jobs:
-  lint_markdown:
+  test:
```

Comment:
if it tests only linting, maybe `test_lint`?

Reply:
It seemed to me that it was appropriate in this way.

I did some fixup to call the JavaScript linter in the `test` task.

Reply:
Ok, 

After giving some thought, I image `lint` would be good as it's the base name of all your scripts

Reply:
In my view, CI calls `npm test` and does not know what it does exactly. Right now it only does linting, but it could do more later 🙂

Reply:
Let's keep `test` then, sorry about that @clementbiron :-)